docs/source/notebook.rst

```diff
@@ -272,13 +272,12 @@ nbconvert.
 Heading cells
 ~~~~~~~~~~~~~
 
-You can provide a conceptual structure for your computational document as a
-whole using different levels of headings; there are 6 levels available, from
-level 1 (top level) down to level 6 (paragraph). These can be used later for
-constructing tables of contents, etc.  As with Markdown cells, a heading
-cell is replaced by a rich text rendering of the heading when the cell is
-executed.
-
+If you want to provide structure for your document, you can use markdown
+heading.  Markdown headings consists of 1 to 6 hash sign ``#`` followed by a
+space and the title of your section. Markdown heading will be converted to
+click able link for each section you can refer to, and will be use as hint for
```

Comment:
The markdown heading will be converted to a clickable link for a section of the notebook. It is also used as a hint when exporting to other document formats, like PDF. We recommend using only one markdown header in a cell and limit the cell's content to the header text. For flexibility of text format conversion, we suggest placing additional text in the next notebook cell.